tests/ZfcRbacTest/Factory/RoutePermissionsGuardFactoryTest.php

```diff
@@ -50,11 +50,16 @@ public function testFactory()
             $this->getMock('ZfcRbac\Service\AuthorizationService', [], [], '', false)
         );
 
-        $pluginManager = new GuardPluginManager();
-        $pluginManager->setServiceLocator($serviceManager);
+        $pluginManager = new GuardPluginManager($serviceManager);
 
         $factory    = new RoutePermissionsGuardFactory();
-        $routeGuard = $factory->createService($pluginManager);
+
+        if (method_exists($serviceManager, 'build')) { // servicemanager v3
+            $routeGuard = $factory->createService($serviceManager);
```

Comment:
as others SMv3 does not call createService() so I don't know what you are testing here